src/main/java/net/dv8tion/jda/internal/audio/AudioConnection.java

```diff
@@ -696,15 +700,18 @@ else if (silenceCounter > -1)
                     else
                         seq++;
 
-                    if (++silenceCounter > 10)
+                    silenceCounter++;
+                    if ((!sentSilenceOnConnect && silenceCounter > 10) || silenceCounter > speakingDelay)
```

Comment:
```suggestion
                    //If we have sent our 10 silent packets on initial connect, or if we have sent enough silent packets
                    // to satisfy the speaking delay, stop transmitting silence.
                    if ((!sentSilenceOnConnect && silenceCounter > 10) || silenceCounter > speakingDelay)
```

Reply:
Took me longer than I want to admit to understand this, thus I add the comment for clarity.